docs/DxcOnLinux.rst

```diff
@@ -0,0 +1,78 @@
+=======================================
+DirectXShaderCompiler on Linux & Mac OS
+=======================================
+
+.. contents::
+   :local:
+   :depth: 3
+
+Introduction
+============
+
+Due to several `requests <https://github.com/Microsoft/DirectXShaderCompiler/issues/1236>`_
+from the community, we have started the effort to enable compilation and running
+of DirectXShaderCompiler on non-Windows platforms (Linux and Mac OS).
+
+
+Building DirectXShaderCompiler
+==============================
+You can follow these steps to build DirectXShaderCompiler on Linux/MacOS:
+
+(We are assuming you have cloned the repository to `~/DirectXShaderCompiler`)
+
+
+.. code:: sh
+
+  cd ~/DirectXShaderCompiler
+  mkdir build && cd build
+  cmake .. -GNinja                                 \
+    $(cat ../utils/cmake-predefined-config-params) \
+    -DSPIRV_BUILD_TESTS=ON                         \
+    -DCMAKE_BUILD_TYPE=${DXC_BUILD_TYPE}           \
+    -DCMAKE_C_COMPILER=${CC} -DCMAKE_CXX_COMPILER=${CXX}
+
+  ninja dxc
+
+
+``DXC_BUILD_TYPE`` is the cmake build type (``Debug`` or ``Release`` or
+``RelWithDebInfo``).
+You can use ``clang`` and ``clang++``, or ``gcc`` and ``g++`` as your C/CXX
+compiler.
+
+You should now have the dxc executable located at `~/DirectXShaderCompiler/build/bin/dxc`.
+
+And you should be able to successfully run commands as you would on Windows, e.g:
+
+.. code:: sh
+
+  ./bin/dxc -help
+  ./bin/dxc -T <target> -E <entry> <InputFile.hlsl>
```

Comment:
Kinda nit-picking, but it's better to keep consistent case style. And I think `.hlsl` is not required.

s/entry/entry-point-name/
s/InputFile.hlsl/input-hlsl-file/

Reply:
Yep. Done.